### ISSUE TITLE
Missing r10k::include_postrun_command?

### ISSUE THREAD

**Issue by esalberg** [acidprime/r10k#137]
The documentation states:
r10k::include_postrun_command: true

However, I see include_prerun_command in init.pp but nothing similar for include_postrun_command.

I had no trouble getting the postrun command setup by putting the following into my code:
  include ::r10k::postrun_command
(However, there's a benefit to the other method in just being able to add the parameter to Hiera.)

I wanted to check and see if this was a documentation issue or an oversight before submitting a pull request.  I can either add r10k::include_postrun_command to match prerun, or I can update the docs.

**Comment by acidprime** [comment]
This is def a bug, we moved from using prerun to postrun for most deployments as it does not prevent catalogs being applied on your master when say someone screws the Puppetfile up. I will add it as a toplevel param in v3 of the module as I am going to remove prerun command.

Can you elaborate on `trouble getting the postrun command setup by putting the following into my code:` with an error message?

**Comment by esalberg** [comment]
Thanks!

That says "no trouble", so no error message. :)  I can leave the include in place as as workaround until the code gets updated - everything's working great so far.

**Comment by acidprime** [comment]
Here is the code I normally use in a PE environment

```puppet
  class {'r10k::postrun_command':
    command => '/opt/puppet/bin/r10k deploy environment -p -v debug'
  }
```

**Comment by acidprime** [comment]
Decided to fix this in the 2.x series as it was documented and thus a bug not a regession 
released in [2.7.0](https://github.com/acidprime/r10k/releases/tag/v2.7.0)

**Comment by acidprime** [comment]
@esalberg thanks for the bug report!
